contrib/julia-config.jl

```diff
@@ -51,6 +51,8 @@ end
 function ldlibs()
     if is_unix()
         return replace("""-Wl,-rpath,$(libDir()) -ljulia""","\\","\\\\")
+    elseif is_windows()
+        return replace("""-ljulia -lopenlibm""","\\","\\\\")
```

Comment:
Yeah, the replacement seems unnecessary. I was just following the pattern that was there before.

The new case is because `-lopenlibm` is only for windows and not, e.g., for OS X.